package.json

```diff
@@ -250,7 +250,7 @@
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-storybook": "^0.9.0",
-    "eslint-plugin-testing-library": "^5.11.0",
+    "eslint-plugin-testing-library": "^5.11.1",
```

Comment:
Update because I need this fix https://github.com/testing-library/eslint-plugin-testing-library/pull/740